scripts/add_uniprot_sequences.py

```
@@ -0,0 +1,143 @@
+    try:
+        print(f"  Fetching {accession}...", end=" ")
+        with urlopen(url) as response:
+            fasta_data = response.read().decode("utf-8")
+            # Parse FASTA format - skip header line, join sequence lines
+            lines = fasta_data.strip().split("\n")
+            sequence = "".join(lines[1:])  # Skip first line (header)
+            print(f"OK ({len(sequence)} aa)")
+            return sequence
+    except HTTPError as e:
+        if e.code == 404:
+            print(f"NOT FOUND")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)